OWML.ModHelper/Logging/ModSocketOutput.cs

```diff
@@ -1,59 +1,111 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Net;
 using System.Net.Sockets;
 using System.Text;
+using Newtonsoft.Json;
 using OWML.Common;
 
 namespace OWML.ModHelper
 {
     public class ModSocketOutput : ModConsole
     {
-        private const string LocalHost = "127.0.0.1";
-        private const string NameMessageSeparator = ";;";
-
+        private int _port;
         private static Socket _socket;
 
         public ModSocketOutput(IOwmlConfig config, IModLogger logger, IModManifest manifest) : base(config, logger, manifest)
         {
             if (_socket == null)
             {
-                CreateSocket();
+                _port = config.SocketPort;
+                ConnectToSocket();
             }
         }
 
-        public override void WriteLine(string s)
+        [Obsolete]
+        public override void WriteLine(params object[] objects)
```

Comment:
This is short and sweet and I no longer see any reason for deprecaring it 😋
You?

Reply:
Eh, I don't care. Deprecate if you like, your call ;-)

Reply:
object[] is kinda just messy to use, and you said we should get rid of it?